tests/predict/test_decode.py

```diff
@@ -12,3 +14,67 @@ def test_mlp_decoder():
     input_batch = torch.randn((16, in_dim))
     output = mlp_decoder(input_batch)
     assert output.size() == (16, 1)
+
+
+def test_linear_classifier_model_shape():
+    in_dim = 10
+    out_dim = 5
+    model = LinearClassifier(in_dim, out_dim)
+    assert model.fc.weight.size() == (out_dim, in_dim)
+
+
+def test_linear_classifier_output_shape():
```

Comment:
Can this be combined with the above?

Reply:
Since these are unit tests, it would be good to test small sections of the module separately for easier debugging.